src/lando/main/admin.py

```diff
@@ -112,7 +142,10 @@ class RevisionAdmin(admin.ModelAdmin):
 
     def revision(self, instance: Revision) -> str:
         """Return a Phabricator-like revision identifier."""
-        return f"D{instance.revision_id}"
+        if instance.revision_id:
+            return f"D{instance.revision_id}"
+
+        return f"{instance.id}"
```

Comment:
Feels like we should always be including the internal revision ID, regardless of what type of revision this is.

Also, this could use the `revision.is_phabricator_revision` property I mentioned earlier.